presto-main/src/main/java/com/facebook/presto/execution/QueryStats.java

```diff
@@ -373,12 +373,18 @@ public static QueryStats create(
                 PlanFragment plan = stageInfo.getPlan().get();
                 for (OperatorStats operatorStats : stageExecutionStats.getOperatorSummaries()) {
                     // NOTE: we need to literally check each operator type to tell if the source is from table input or shuffled input. A stage can have input from both types of source.
+                    // NOTE: also check for operator names from presto native execution.
                     String operatorType = operatorStats.getOperatorType();
-                    if (operatorType.equals(ExchangeOperator.class.getSimpleName()) || operatorType.equals(MergeOperator.class.getSimpleName())) {
+                    if (operatorType.equals(ExchangeOperator.class.getSimpleName()) ||
```

Comment:
It would be nice to declare a constant set of operator names and use it here. It would also be helpful to declare 2 separate sets: one for Java and another for Native execution and choose which one to use based on isNativeExecution flag.

Reply:
@mbasmanova 
Few Qs regarding the comment.
1. Where to declae such constants? The same java file?
2. Should we declare constants for java names too? It seems using the class names and does not really need constants.
3. If we use isNativeExecution flag, then the code won't work until we change presto_cpp. But I guess this would be short term and is ok.

Reply:
We can define constants in this java file.

We can initially include both Java names and Velox names in the map of operators used when isNativeExecution == true.